fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/SavingsAccountsApiResourceSwagger.java

```diff
@@ -233,6 +235,33 @@ private PostSavingsAccountsRequest() {}
         public String submittedOnDate;
         @Schema(example = "123")
         public String externalId;
+        @Schema(example = "5.0")
```

Comment:
Seems unrelated change... should it be extracted into a separate PR?

Reply:
Please dont use `Double`. It is either whole number and Integer / Long or BigDecimal to support fractions.

Reply:
Please dont use Double types...